Source/WebCore/css/typedom/StylePropertyMap.h

```diff
@@ -31,7 +31,25 @@ namespace WebCore {
 
 class StylePropertyMap : public StylePropertyMapReadOnly {
 public:
-    virtual void clearElement() = 0;
+    ExceptionOr<RefPtr<CSSStyleValue>> get(ScriptExecutionContext&, const AtomString& property) const override;
+    ExceptionOr<Vector<RefPtr<CSSStyleValue>>> getAll(ScriptExecutionContext&, const AtomString&) const override;
+    ExceptionOr<bool> has(ScriptExecutionContext&, const AtomString&) const override;
```

Comment:
Can they be final?

Reply:
Had to make some changes to one of the child classes but yes.